### PR TITLE
list images in order on index

### LINKED ISSUE #3: Image Index
#### As a user I want the homepage to display all saved images.

__Story__:

Now that we are saving images for our users, we want them to be able to view
the list of images in the system. You might be thinking, "what if my users only
want to view images of cats", or "what if my users only want to see the images
they uploaded". Those are great questions. Maybe you will have the opportunity
to add those features in a future story. For now, we will keep it simple and
quickly deliver a little more value to our users.

__Acceptance criteria__:
- [ ] New images that are added show up on the homepage.
- [ ] These images are persisted if the browser is closed or even if the
  server is restarted.
- [ ] Images are not displayed wider than 400px.
- [ ] Newest images appear first.

__Dependencies__:
- Save Image Link

### PR DESCRIPTION
**Pull request by seanhelvey**
- Added method on model to sort images in ascending order by created date
- Display all images on index with 400px max-width
- Tests for route, controller, and model

closes #3 

img

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,7 @@
  *= require_tree .
  *= require_self
  */
+
+img {
```
Comment (grahamotte) [inline app/assets/stylesheets/application.css]: We use [BEM](http://getbem.com/introduction/) for our css, would to good to use here as well as opposed to any `img`

File: app/models/image.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class Image < ApplicationRecord
   validates :url, presence: true, null: false, format: { with: URI.regexp(%w[http https]) }
+
+  scope :recent, -> { order(created_at: :desc) }
```
Comment (grahamotte) [inline app/models/image.rb]: Nice scope!

File: test/models/image_test.rb [hunk 3]
```diff
@@ -22,4 +22,11 @@ def test_image__invalid_with_bad_url
     image = Image.new(url: 'test')
     assert_not image.valid?
   end
+
+  def test_image__recent_in_order
+    older_image = Image.create(url: 'https://www.w3schools.com/w3css/img_lights.jpg')
+    newer_image = Image.create(url: 'https://www.w3schools.com/w3css/img_lights.jpg')
+
+    assert_equal Image.recent.to_a.first(2), [newer_image, older_image]
```
Comment (grahamotte) [inline test/models/image_test.rb]: when using `assert_equal`, you want to put your static, expectation value as the first param and the calculated one as the second. This just makes the failure message make sense

Comment (seanhelvey) [reply test/models/image_test.rb]: Thank you! Got it backwards there.

File: app/models/image.rb (hunk 2 above)
Comment (seanhelvey) [reply app/models/image.rb]: Why thank ya!

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (seanhelvey) [reply app/assets/stylesheets/application.css]: Got it. Done!

File: app/views/images/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<% @images.each do |image| %>
+  <%= image_tag(image.url, :class => "img") %>
```
Comment (grahamotte) [inline app/views/images/index.html.erb]: This doesn't really make a difference, but here you can use the shorter hash syntax `class: 'img'` when the key is a symbol.

**Pull request merged**